Pod/Classes/ObjC/URBNValidationRulesCompat.swift

```diff
@@ -115,13 +115,13 @@ import Foundation
     
     /// Here we're making sure that we're given an NSDate because objc is 
     /// dumb and will allow the user to not give an NSDate
-    public override func validateValue(value: AnyObject?) -> Bool {
-        guard let value = value as? NSDate else { return false }
-        return super.validateValue(value)
+    open override func validateValue(_ value: AnyObject?) -> Bool {
+        guard let value = value as? Date else { return false }
+        return super.validateValue(value as AnyObject?)
     }
     
-    public override func validateValue(value: AnyObject?, key: String) -> Bool {
-        guard let value = value as? NSDate else { return false }
-        return super.validateValue(value, key: key)
+    open override func validateValue(_ value: AnyObject?, key: String) -> Bool {
+        guard let value = value as? Date else { return false }
+        return super.validateValue(value as AnyObject?, key: key)
```

Comment:
Sorry, I wasn't trying to be flippant. Casting to AnyObject in this case did exactly what it did implicitly before, but most casting in swift 3 is required to be explicit (which is annoying).

I'll look into it more to see if it makes sense in this particular case